pollination/honeybee_vtk/post_process.py

```diff
@@ -0,0 +1,78 @@
+from dataclasses import dataclass
+from pollination_dsl.function import Function, command, Inputs, Outputs
+
+
+@dataclass
+class Gif(Function):
+    """Export a GIF from a folder of time step images."""
+
+    time_step_images_folder = Inputs.folder(
```

Comment:
I think the input name can be simple and you can add more to the description. What about `input_folder` and then with a better description that explains that what other Function generates this folder?